### ISSUE TITLE
Data is 2 days out of date

### ISSUE THREAD

**Issue by amnonbc** [amnonbc/covid19#3]
It might be worth replacing https://covid.ourworldindata.org/data/full_data.csv with a more regularly updated source.

**Comment by amnonbc** [comment]
see https://ourworldindata.org/covid-sources-comparison
Why we stopped relying on data from the World Health Organization

Up until 18 March 2020, we relied on the World Health Organization (WHO) as our only data source. The WHO publishes daily Situation Reports and went through them every day to convert these data publications of the WHO – single PDF files with printed tables – into consistent data files. We undertook the extra effort to bring the data from the Situation Reports into the right format to be able to rely on the international agency of the UN with the mandate to provide data on the COVID-19 outbreak.

Unfortunately, on 18 March 2020, Situation Report 58 shifted the reporting cutoff time. With this change by the WHO comparability with earlier figures was compromised. There is now an overlap between the last two WHO data releases (Situation Reports 57 and 58) – the hours between midnight and 9am (CET) are reported in both reports due to the cutoff time change.
